### PR TITLE
Add Troubleshooting guide for jobs

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

**Comment by hao1939** [comment]
It's great to have the feature 'Simulate environment locally'!
Before that, I usually run a job 'sleep infinity' for debugging.

File: docs/user/troubleshooting_job.md [hunk 1]
```diff
@@ -17,51 +17,197 @@
...
+
+## Diagnostic issues
+
+### Job is waiting for hours
+
+In general, jobs of OpenPAI stays in waiting status about 1 minute. But if there is not enough resource, a job may stay in waiting status longer.
+
+![waiting](imgs/web_job_list_waiting.png)
+
+In this case, the job is in queue to wait resource. When some jobs complete, this job will get resource to run. Another way is to reduce requested resource, so this job may get enough resource easier.
```
Comment (xudifsd) [inline docs/user/troubleshooting_job.md]: Can direct user to virtual cluster page to let user checks if requesting resource can not be satisfied by this VC.

Comment (xudifsd) [reply docs/user/troubleshooting_job.md]: Also #2208 will be useful for user to debug.

File: docs/user/troubleshooting_job.md [hunk 2]
```diff
@@ -17,51 +17,197 @@
...
+
+- Click *Go to Job Metrics Page* as below in job details page, if some tasks are assigned IP, and ports.
+
+![job link](imgs/web_job_details_metrics.png)
+
+- A new page is opened and show metrics of this job.
+
+![job link](imgs/web_job_metrics.png)
+
+- The *memory usage*, *network traffic* and *disk bandwidth* uses absolute value. It's easy to understand.
```
Comment (xudifsd) [inline docs/user/troubleshooting_job.md]: Should notice: PAI sacrifices network metric accuracy by using host network in docker container for better performance, so, PAI can not relay on docker virtual bridge to get accurate network traffic consumption.

PAI uses another way: get all connections and traffic used by each connection, then break down into each container. This logic is implemented in [job-exporter](https://github.com/Microsoft/pai/blob/master/src/job-exporter/src/network.py). So, if your connection is short-lived or not happened in container namespace(for example, NFS mount), traffic in this connection will not captured by job-exporter and hence will show 0 in panel.

File: docs/user/troubleshooting_job.md (hunk 1 above)
Comment (squirrelsc) [reply docs/user/troubleshooting_job.md]: VC is an advanced topic for admin. A cluster may enable or disable preemption, it's hard to let user to judge it, and even in the virtual cluster, fragments resource still blocks allocation. The goal is to let user know fragmentation may block resource allocation, that's enough to a user.

File: docs/user/troubleshooting_job.md (hunk 2 above)
Comment (squirrelsc) [reply docs/user/troubleshooting_job.md]: Thanks, I will update it accordingly.

File: docs/user/troubleshooting_job.md (hunk 1 above)
Comment (xudifsd) [reply docs/user/troubleshooting_job.md]: Ok, maybe we can change the doc later when #2208 has finished.

**Pull request merged**